solar_crypto/constants.py

```diff
@@ -1,25 +1,25 @@
 from enum import Enum
 
-TRANSACTION_TRANSFER = 0
+TRANSACTION_LEGACY_TRANSFER = 0
 TRANSACTION_SECOND_SIGNATURE_REGISTRATION = 1
 TRANSACTION_DELEGATE_REGISTRATION = 2
 TRANSACTION_VOTE = 3
 TRANSACTION_MULTI_SIGNATURE_REGISTRATION = 4
 TRANSACTION_IPFS = 5
-TRANSACTION_MULTI_PAYMENT = 6
+TRANSFER = 6
```

Comment:
Should be called `TRANSACTION_TRANSFER` right?

Reply:
Yes, will fix. Thanks